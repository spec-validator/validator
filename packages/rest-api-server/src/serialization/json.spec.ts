import { JsonSerialization } from '.'

const json = new JsonSerialization()


test('serialize', () => {
  expect(json.serialize('html')).toMatchSnapshot()
})

test('deserialize', () => {
  expect(json.deserialize('"html"')).toMatchSnapshot()
})
