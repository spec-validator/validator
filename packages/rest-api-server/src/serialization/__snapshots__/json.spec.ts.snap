// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`deserialize 1`] = `"html"`;

exports[`serialize 1`] = `"\\"html\\""`;
